frontend/src/pages/SettingsRouter/SettingsRouter.tsx

```diff
@@ -180,6 +213,27 @@ export const SettingsRouter = () => {
 							</NavLink>
 						))}
 					</Stack>
+					<Stack gap="0">
+						<Box mt="12" mb="10" ml="8">
```

Comment:
@jay-khatri @julian-highlight does it make sense to have the left panel organized as `Account Settings`, `Workspace Settings`, `Project Settings` instead? more of a heirarchy of what belongs to what